mm2src/coins/utxo/bch.rs

```diff
@@ -824,7 +824,15 @@ impl UtxoCommonOps for BchCoin {
         gas_fee: Option<u64>,
         stage: &FeeApproxStage,
     ) -> TradePreimageResult<BigDecimal> {
-        utxo_common::preimage_trade_fee_required_to_send_outputs(self, outputs, fee_policy, gas_fee, stage).await
+        utxo_common::preimage_trade_fee_required_to_send_outputs(
+            self,
+            &self.ticker().to_string(),
```

Comment:
It's better to take `&str` instead of `&String`. Then you can avoid casting `self.ticker(): &str` to `String`.